test/verify/check-machines

```diff
@@ -2973,6 +2972,7 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         b.wait_present(".modal-dialog")
         b.click(".modal-footer button:contains(Remove)")
         b.wait_present(".modal-footer .spinner")
+        wait(lambda: "vdc" not in m.execute("virsh domblklist subVmTest1"), delay=3)
```

Comment:
Please add a comment what this is for, and document the "why" instead of the "what" in the commit log. The statement below already waits for vdc go go away from the UI. Does that simply need a longer timeout? You can use `with b.wait_timeout(180):` and add a comment why this is slow.

Reply:
Ack so I will remove this virsh cmdline check the simply increase the timeout in the UI check.